server/src/test/java/org/elasticsearch/index/mapper/BinaryFieldMapperTests.java

```diff
@@ -121,9 +123,12 @@ public void testStoredValue() throws IOException {
 
     @Override
     protected Object generateRandomInputValue(MappedFieldType ft) {
-        assumeFalse("We can't parse the binary doc values we send", true);
-        // AwaitsFix https://github.com/elastic/elasticsearch/issues/70244
-        return null;
+        if (rarely()) {
+            return null;
+        } else {
+            byte[] value = randomAlphaOfLengthBetween(1, 30).getBytes(StandardCharsets.UTF_8);
```

Comment:
I think it'd be nice to also try the case where these aren't utf-8 bytes. Just some byte array of pure random stuff.

Reply:
Great comment! I guess it makes sense just always use some random bytes, I guess this is what Binary field is intended for – to store random binary values.

Reply:
Addressed in 3aa1286d84fe1628740d286e77e7314a92dbe9e4